plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -66,6 +65,18 @@ setup_memory_allocation()
   pcre_free   = &ts_free;
 }
 
+static inline bool
```

Comment:
Since C++11, static or inline both do the same thing for functions, restrict their scope to the compilation unit.  Compilers can inline functions defined in the compilation unit even if they don't have the 'inline' keyword.  I don't think any current major compiler even uses the inline keyword as a hint that the function should actually be inlined.

Remember you can also use anonymous namespaces to scope things in the compilation unit (although this can be inconvenient for debugging).